ios/src/ImageCropPicker.h

```diff
@@ -24,31 +26,7 @@
 #import "RCTImageLoaderProtocol.h"
 #endif
 
-#if __has_include("QBImagePicker.h")
-#import "QBImagePicker.h"
-#elif __has_include(<QBImagePickerController/QBImagePickerController.h>)
-#import <QBImagePickerController/QBImagePickerController.h>
-#elif __has_include("QBImagePickerController.h") // local QBImagePickerController subspec
-#import "QBImagePickerController.h"
-#else
-#import
-#import "QBImagePicker/QBImagePicker.h"
-#endif
-
-
-#import <TOCropViewController/TOCropViewController.h>
-
-#import "UIImage+Resize.h"
-#import "UIImage+Extension.h"
-#import "Compression.h"
-#import <math.h>
-
-@interface ImageCropPicker : NSObject<
-UIImagePickerControllerDelegate,
-UINavigationControllerDelegate,
-RCTBridgeModule,
-QBImagePickerControllerDelegate,
-TOCropViewControllerDelegate>
+@interface ImageCropPicker : NSObject<UINavigationControllerDelegate, RCTBridgeModule>
```

Comment:
The publicly visible interface doesn't need to expose the fact that it conforms to all these protocols